Makefile

```
@@ -281,7 +287,7 @@ ci-local-coverage: fmt-check lint coverage-summary docs build ## Alias: run loca
 all: ci coverage bench-check ## Full validation suite
 
 .PHONY: release-check
-release-check: ## Check release readiness
+release-check: package-check ## Check release readiness
```

checkmake (0.3.2)

[warning] Target body for "release-check" exceeds allowed length of 5 lines (6).

(maxbodylength)